tests/unit/mocks/module_mock.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2023, Denys Denysyev (ddenysyev@purestorage.com)
+# GNU General Public License v3.0+ (see COPYING.GPLv3 or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from unittest.mock import MagicMock
+
+
+class ModuleSucceeded(Exception):
+    pass
+
+
+class ModuleFailed(Exception):
+    pass
+
+
+class ModuleMock(MagicMock):
+    def __init__(self, params, check_mode=False, exit_json=None, fail_json=None):
+        super(ModuleMock, self).__init__()
```

Comment:
What does this do? Isn't the following the same?
```suggestion
        super().__init__()
```